lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -90,9 +104,19 @@ class IRFFITSWriter(Tool):
         -g /path/to/DL2_MC_gamma_file.h5
         -o /path/to/irf.fits.gz
         --point-like (Only for point_like IRFs)
-        --fixed-gh-cut 0.9
-        --fixed-theta-cut 0.2
+        --global-gh-cut 0.9
+        --global-theta-cut 0.2
         --irf-obs-time 50
+
+    Or use energy-dependent cuts based on a gamma efficiency:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+        --energy-dependent-gh
+        --energy-dependent-theta
```

Comment:
Is there an advantage in having these? It would be enough to activate the E-dependent cuts if the --gh-efficiency and/or --theta-containment options below are used, right?

Reply:
Yes, but this would need to change the defaults of the configuration to have them all `None` and activate the corresponding setting by giving it.

And I think there is a bug in traitlets that doesn't allow a `Float(allow_none=True)`:

https://github.com/ipython/traitlets/issues/637

I fixed it for `ctapipe_io_lst` here:

https://github.com/cta-observatory/ctapipe_io_lst/blob/e78acdc43ef4c4c95c83ec5100b7c6373af9df86/ctapipe_io_lst/event_time.py#L26

Reply:
The config file may have the options, but it will be up to the user to pass these separate flags to use the energy-dependent cuts

Reply:
Yes, we understand the current situation. We are proposing to change that.

The issue seems to be fixed in traitlets 5.1 we are using now.